uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/LeaveRequestCalendarFeedConfig.php

```diff
@@ -251,14 +251,7 @@ public function addSelectWhereClause() {
       return;
     }
 
-    $accessibleDepartments = $this->getAccessibleDepartmentsForLoggedInContact();
-    $accessibleLocations = $this->getAccessibleLocationsForLoggedInContact();
-
-    $inaccessibleFeedConfigs = $this->getNotAccessibleFeedConfigs(
-      $limitedVisibilityFeedConfigs,
-      $accessibleDepartments,
-      $accessibleLocations
-    );
+    $inaccessibleFeedConfigs = $this->getInAccessibleFeedConfigsForCurrentUser($limitedVisibilityFeedConfigs);
```

Comment:
Sorry, one minor thing - `getInAccessible` isn't "Get In Accessible" should be 'getInaccessibleFeedConfigsForCurrentUser`